bindata/v4.1.0/config/defaultconfig.yaml

```diff
@@ -79,8 +79,6 @@ apiServerArguments:
   # switch to direct pod IP routing for aggregated apiservers to avoid service IPs as on source of instability
   enable-aggregator-routing:
     - "true"
-  enable-logs-handler:
-    - "false"
```

Comment:
didn't we have a carry patch?

Reply:
@sttts carry to add this back/leverage it?

Reply:
yes. Maybe not. I thought @smarterclayton pushed us to keep this working in the past.